zerver/openapi/zulip.yaml

```diff
@@ -11684,8 +11684,8 @@ paths:
                         description: |
                           Present if `realm_incoming_webhook_bots` is present in `fetch_event_types`.
 
-                          An array of dictionaries where each dictionary describes a type of incoming webhook
-                          integration that is available to be configured on this Zulip server.
+                          An array of dictionaries where each dictionary describes a type of non-legacy incoming
+                          webhook integration that is available to be configured on this Zulip server.
```

Comment:
This probably doesn't require an API feature level update or API documentation change -- we're just changing the meaning of "available to be configured on this server"?

This commit's comment message include an explanation for why we're excluding these, and possibly also that should go in a comment in `zerver/lib/events.py`.